### PR TITLE
Allow users to choose Password or LDAP login

### LINKED ISSUE #118: Make auth-type an option
Put a config option for auth-type to send to pymesync. The default should be LDAP, but the interface should have an option to do password as well.

### PR DESCRIPTION
**Pull request by aaroncohen73**
Fixes #118 
## Changes in this PR.

- [X] Added auth_type selector to login page
## Testing this PR.

1. Run `nosetests`
2. Set `TIMESYNC_URL = https://timesync.osuosl.org/v0` in config.py
3. Run the app with `docker-compose up` and go to `/login`
4. Click on the select box and choose `LDAP`, then log in with your LDAP credentials
5. See that you've been logged in
### Expected Output.

```
14:19 $ nosetests
......................................................................................................................
----------------------------------------------------------------------
Ran 118 tests in 1.795s

OK
```

@osuosl/devs

### CODE REVIEW

File: app/views/login.py [hunk 1]
```diff
@@ -13,13 +13,15 @@ def login():
     if form.validate_on_submit():
         username = form.username.data
         password = form.password.data
+        auth_type = form.auth_type.data
+        print auth_type
```
Comment (Kennric) [inline app/views/login.py]: extraneous print?

File: app/forms.py [hunk 2]
```diff
@@ -9,6 +9,9 @@ class LoginForm(Form):
                            render_kw={'placeholder': 'username'})
     password = PasswordField('password', validators=[DataRequired()],
                              render_kw={'placeholder': 'password'})
+    auth_type = SelectField('Authentication Type', validators=[DataRequired()],
+                            choices=[('password', 'Password'),
```
Comment (Kennric) [inline app/forms.py]: I would like to make ldap the default.

**Comment by MaraJade** [comment]
+1

**Comment by subnomo** [comment]
Looks good. :+1:

**Pull request merged**